tests/components/zwave_js/test_config_flow.py

```diff
@@ -383,6 +411,96 @@ async def test_abort_discovery_with_existing_entry(
     assert entry.data["url"] == "ws://host1:3001"
 
 
+async def test_abort_hassio_discovery_with_existing_flow(
+    hass, supervisor, addon_options
+):
+    """Test hassio discovery flow is aborted when another discovery has happened."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
```

Comment:
We don't need this since we have a fixture that sets it up and is set to autouse.

Reply:
Excellent. Less code 👍